EIPS/eip-2770.md

```diff
@@ -0,0 +1,178 @@
+---
+eip: 2770
+title: Meta-Transactions Forwarder Contract
+author: Dror Tirosh <dror.tirosh@gmail.com>, Alex Forshtat <forshtat1@gmail.com>
```

Comment:
```suggestion
author: Alex Forshtat (@forshtat) Dror Tirosh <dror.tirosh@gmail.com>, Alex Forshtat <forshtat1@gmail.com>
```
```suggestion
author: Alex Forshtat (@forshtat) Dror Tirosh <dror.tirosh@gmail.com>
```
Either is acceptable, but at least one of the authors should be reachable on GitHub.